### PR TITLE
HEXDEV-782: Remove dependencies from download page

### PR DESCRIPTION
**Pull request by jefffohl**
This PR addresses HEXDEV-782 - remove un-needed dependencies and cleanup the downloads page a bit. Some things addressed:

- Remove dependency on clipboard.js - use native browser API for this instead.
- Remove dependency on jQuery
- Remove un-needed stylesheets - use unicode clipboard emoji for copy icon: https://unicode-table.com/en/1F4CB/

Here is how it looks on a Mac:

Screen Shot 2022-09-17 at 8 30 18 AM

There is a lot more that could be done to clean up the page, such as separate out JS and CSS into separate files, and align the branding of the page to the current branding of H2O, but that can be left to later work if it is desired.

### CODE REVIEW

File: h2o-dist/index.html [hunk 1]
```diff
@@ -8,10 +8,8 @@
     </title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    <link rel="stylesheet" href="primer.css">
```
Comment (jefffohl) [inline h2o-dist/index.html:12]: This file is no longer needed.

File: h2o-dist/index.html [hunk 2]
```diff
@@ -433,75 +517,6 @@ <h1><span>H<sub>2</sub>O</span></h1>
           <span id = "quickstart-maven-tab">Use from Maven</span>
           <span id = "quickstart-kubernetes-tab">Kubernetes</span>
         </div>
-        <!-- Clipboard Icon !-->
```
Comment (jefffohl) [inline h2o-dist/index.html:436]: No longer a need for clipboard.js, octicons, or jQuery

File: h2o-dist/index.html [hunk 3]
```diff
@@ -761,4 +755,44 @@ <h2>Booklets</h2>
       </div>
     </div>
   </body>
+  <script type="text/javascript">
```
Comment (jefffohl) [inline h2o-dist/index.html:758]: By putting the script at the end of the page, we can avoid using jQuery to tell us when the DOM is ready.

File: h2o-dist/index.html [hunk 4]
```diff
@@ -761,4 +755,44 @@ <h2>Booklets</h2>
...
+          if (!targetId) {
+              console.error('No targetId found');
+              return;
+          }
+          const text = document.getElementById(targetId.replace('#',''))?.innerText;
+          if (!text) {
+              console.error(`No copyable element or text found for ID ${targetId}`);
+              return;
+          }
+          navigator.clipboard.writeText(text).then(() => {
```
Comment (jefffohl) [inline h2o-dist/index.html:779]: We can use the browser's native API for working with the clipboard here.

File: h2o-dist/index.html [hunk 5]
```diff
@@ -164,6 +162,92 @@
...
+      }
+
+      .btn.copy_button:focus, .btn.copy_button:active {
+          border-color: #4c8b36;
+          background: #fff;
+      }
+
+      .copy_button:before {
+          position: absolute;
+          content: '\1F4CB';
```
Comment (jefffohl) [inline h2o-dist/index.html:215]: This is where we define the unicode clipboard icon

**Comment by michalkurka** [comment]
Thank you, Jeff! This looks great

**Pull request merged**